### DISCUSSION TITLE
make ucm/event/event.h installed

### DISCUSSION THREAD

**Discussion by skypexu** [openucx/ucx#8641] [Ideas]
It would be nice to install ucm/event/event.h when make install, this would allow us to unregister memory when we do know that 
the memory is released.

**Comment by yosefe** [comment]
it can be done by following API in ucm.h:
```
/**
 * @brief Call the handlers registered for aggregated VM_MUNMAP event.
 */
void ucm_vm_munmap(void *addr, size_t length);
```

**Reply by skypexu** [reply]
> it can be done by following API in ucm.h:
> 
> ```
> /**
>  * @brief Call the handlers registered for aggregated VM_MUNMAP event.
>  */
> void ucm_vm_munmap(void *addr, size_t length);
> ```

but the memory is not allocated by mmap,  it is from third party library.

**Reply by yosefe** [reply]
> but the memory is not allocated by mmap, it is from third party library.

it doesn't matter; Calling this function with dispatch memory release event to all consumers.

**Reply by skypexu** [reply]
OK, I found the API, thanks!
